improver_tests/calibration/reliability_calibration/test_ApplyReliabilityCalibration.py

```diff
@@ -82,16 +83,19 @@ def setUp(self):
                 [0, 0, 250, 500, 750],  # Observation count
                 [0, 250, 500, 750, 1000],  # Sum of forecast probability
                 [1000, 1000, 1000, 1000, 1000],  # Forecast count
-            ]
+            ],
+            dtype=np.float32,
         )
+
         # Under forecasting exceeding 280K.
         reliability_data_1 = np.array(
             [
                 [250, 500, 750, 1000, 1000],  # Observation count
                 [0, 250, 500, 750, 1000],  # Sum of forecast probability
                 [1000, 1000, 1000, 1000, 1000],  # Forecast count
-            ]
-        )
+            ],
+            dtype=np.float32,
+        )  # Forecast count
```

Comment:
Not sure this "Forecast count" should be here.

Reply:
I've corrected this.